addon/components/async-view.js

```diff
@@ -87,14 +87,11 @@ let AsyncView = Component.extend({
    * @type {any}
    * @public
    */
-  fulfilledValue: computed('value', 'value.isPending', 'value.content',
-                           function()
-  {
+  fulfilledValue: computed('value', 'value.isPending', 'value.content', function() {
     let value = get(this, 'value');
-
-    return value && typeof value.isPending !== 'undefined' ?
-            (typeof value.content !== 'undefined' ? value.content : undefined) :
-            value;
+    return value && typeof value.get('isPending') !== 'undefined' ?
+      (typeof value.content !== 'undefined' ? value.content : undefined) :
```

Comment:
Are all properties computed? If so, shouldn't these also be changed to `value.get('content')`?